mozdownload/scraper.py

```diff
@@ -24,21 +24,35 @@
 from mozdownload.timezones import PacificTimezone
 from mozdownload.utils import urljoin
 
-APPLICATIONS = ('devedition', 'firefox', 'fennec', 'thunderbird')
+APPLICATIONS = ('devedition', 'firefox', 'fenix', 'thunderbird')
 
 # Some applications contain all locales in a single build
-APPLICATIONS_MULTI_LOCALE = ('fennec')
+APPLICATIONS_MULTI_LOCALE = ('fenix')
 
-# Used if the application is named differently than the subfolder on the server
-APPLICATIONS_TO_FTP_DIRECTORY = {'fennec': 'mobile'}
-# Used if the application is named differently then the binary on the server
+# Used if the application is named differently than the binary on the server
 APPLICATIONS_TO_BINARY_NAME = {'devedition': 'firefox'}
 # Used when sorting versions
 APPLICATIONS_TO_VERSION_CLASS = {'devedition': 'DeveditionVersion',
                                  'firefox': 'FirefoxVersion',
```

Comment:
I still see a problem with downloading here when using `mozdownload -a fenix -v latest -p android-x86_64`. In this case we do not have a mapper from fenix to a version class. I had a look at mozilla-version but there is actually no specific class for Fenix/GeckoView. We cannot use FennecVersion because that's a different product. 

I'll sort that out with the maintainers, so could you for now please add a mapping from `fenix` to `FirefoxVersion`? Thanks.

Reply:
I've created https://github.com/mozilla-releng/mozilla-version/pull/105 to more easily can import `MobileVersion`. Unless that has been merged and released, lets use `FirefoxVersion` as mentioned above.

Reply:
This PR has been merged now. But we have to wait for the next release. Given that I don't know when this will happen please use `FirefoxVersion` as proposed and I'll change it later on.

Reply:
added the fix

Reply:
This looks great now. Thanks!